src/twisted/mail/relaymanager.py

```diff
@@ -412,7 +414,7 @@ def addMessage(self, message):
         if message not in self.relayed:
             self.waiting[message] = 1
             if self.noisy:
-                log.msg('Set ' + message + ' waiting')
+                log.msg(b'Set ' + message + b' waiting')
```

Comment:
I think it would be better to convert `message` to unicode  (e.g. by `repr` ?) than to emit the log text as a bytestring.